library/ui-styles/src/main/res/values/theme_light.xml

```diff
@@ -142,6 +142,9 @@
         <item name="vctr_keyword_style">@style/Widget.Vector.Keyword</item>
 
         <item name="vctr_toast_background">@color/vctr_toast_background_light</item>
+
+        <!-- Location sharing -->
+        <item name="vctr_live_location">@color/vctr_live_location_dark</item>
```

Comment:
I think you mean `vctr_live_location_light` (even if the actual value is the same)

Reply:
Yes exactly, thanks! I missed that, I will fix this.